### DISCUSSION TITLE
How to Build ImageMagick from Source Code

### DISCUSSION THREAD

**Discussion by Liujiahao0** [ImageMagick/ImageMagick#6487] [Help]
### ImageMagick version

7.

### Operating system, version and so on

ubuntu20.04

### Description

git clone https://github.com/ImageMagick/ImageMagick.git
cd ImageMagick
./configure --prefix="$HOME/ImageMagick/build/"
make
sudo make install
I used the above command to build ImageMagick, but after the construction was completed, the various commands of the program are unusable. I don't know where the problem lies. If there are any issues with my build process, please let me know how to modify it. Thank you.

**Comment by fmw42** [comment]
I suggest you get your Imagemagick from the official distribution at https://imagemagick.org/script/install-source.php#linux rather than GIT.
